coops-ui/src/App.vue

```diff
@@ -117,18 +117,18 @@ export default {
         Promise.all([
           this.getBusinessInfo(businessId),
           axios.get(businessId),
-          axios.get(businessId + '/tasks'),
-          axios.get(businessId + '/filings'),
+          // axios.get(businessId + '/tasks'),
```

Comment:
If this was just for testing, please ensure you restore this code before it gets merged!

Reply:
Absolutely, thanks for reminder. There was a pending update on the backend with regards to these filings and the seed file. Should be good to uncomment!